README.md

```diff
@@ -17,25 +17,36 @@ For more information on `auto-update` sub-directories, please see their own READ
 
 ## Server configuration
 
-The nginx modules are controlled by directives specified in configuration files.
-In our case, these configuration files are stored in a directory named after the configured domain name.  
-For example: `www.openfisca.fr`'s configuration file is `openfisca-ops/www.openfisca.fr/www.openfisca.fr.conf`
+The nginx configuration files are loaded from the `/etc/nginx/sites-available` directory.
+In order to keep these files versionned, we use symlinks from this directory towards the server local clone of this repository.
+
+When you edit one of these files, run this command afterwards so that your changes are taken into account:
+```sh
+service nginx reload
+```
 
 ## Services
 
-We use [systemd](https://wiki.debian.org/systemd) services to ensure the continued operation of the OpenFisca related applications (OpenFisca-France [web](https://api.openfisca.fr/) [APIs](https://api-test.openfisca.fr/), [legislation explorer](https://legislation.openfisca.fr/)). `systemd` is available on most recent (> 2015) linux distributions.
+We use [systemd](https://wiki.debian.org/systemd) services to ensure the continued operation of the OpenFisca related applications (OpenFisca-France [web API](https://fr.openfisca.org/api/v2), [legislation explorer](https://legislation.openfisca.fr/)). `systemd` is available on most recent (> 2015) linux distributions.
+
+The service configuration files are loaded from the `/etc/systemd/system` directory.
+In order to keep these files versionned, we use symlinks from this directory towards the server local clone of this repository.
 
-The service configuration files are loaded from the `/etc/systemd/system` directory. Symlinks pointing to the source code directory can be used.
 
 When you edit one of these files, run this command afterwards so that your changes are taken into account:
 ```sh
 systemctl daemon-reload
 ```
 
+
+## Set up a SSL certificate
+
+See the [dedicated page](guides/Set-up-SSL.md).
+
 ## Renew SSL certificates
 
-To renew the SSL certificate of an OpenFisca related application, run in `root` the following command, replacing `api-test.openfisca.fr` by the domain that needs a certificate update:
+To renew the SSL certificate of an OpenFisca related application, run in `root` the following command, replacing `fr.openfisca.org` by the domain that needs a certificate update:
```

Comment:
> Should we put a reminder somewhere of when the certificate expire ? (in the calendar ?)

Reply:
It expires every 3 months, so I'd definitely not put it here on a repository.

We can put it in the calendars, but they are many certificates, and it's not that obvious to get all their expiration dates.

Normally, Uptime robot will soon warn us before a certificate expires (new feature released yesterday).
We should also consider using certificate auto-renewal : https://github.com/openfisca/openfisca-ops/issues/14